alacritty/src/config/bindings.rs

```diff
@@ -1023,7 +981,9 @@ impl<'a> Deserialize<'a> for RawBinding {
                                     },
                                 },
                                 None => {
-                                    key = Some(Key::deserialize(value).map_err(V::Error::custom)?);
+                                    let k =
+                                        BindingKey::deserialize(value).map_err(V::Error::custom)?;
+                                    key = Some(k);
```

Comment:
I'd assume this was done in an attempt to improve formatting? Seems like it didn't work out, might as well inline it.